addons/account/models/account_account.py

```diff
@@ -489,6 +489,29 @@ def _compute_account_group(self):
         for account in accounts_with_code:
             account.group_id = group_by_code[account.code]
 
+    def _search_account_id_per_group(self, filter_name=''):
```

Comment:
Why making it a distinct function in community ? Shouldn't we just put that in the report, where it's needed, then ? 

Also, if we want to keep it as a function, its name should be better. This function does not "search", so it shouldn't be called that way.